src/Drupal/Commands/pm/PmCommands.php

```diff
@@ -169,7 +169,18 @@ public function validateEnableModules(CommandData $commandData): void
     public function uninstall(array $modules): void
     {
         $modules = StringUtils::csvToArray($modules);
-        $list = $this->addUninstallDependencies($modules);
+
+        $enabled_modules = array_filter($modules, function ($module) {
+            return $this->getModuleHandler()->moduleExists($module);
+        });
+        if ($enabled_modules === []) {
+            throw new \Exception(dt('The following module(s) are not enabled: !list. No modules to uninstall.', ['!list' => implode(', ', $modules)]));
```

Comment:
I think the right language now is not enabled/disabled but installed/uninstalled. We just did a PR to change the from:enable to pm:install

Reply:
Changed